frontend/src/components/form/inputs/Interpreter.tsx

```diff
@@ -1,28 +1,46 @@
-import React from 'react'
+import React from 'react';
 
-import AddIcon from '@material-ui/icons/Add'
-import CloseIcon from '@material-ui/icons/Close'
-import IconButton from '@material-ui/core/IconButton'
-import Hidden from '@material-ui/core/Hidden'
-import Grid from '@material-ui/core/Grid'
-import Typography from '@material-ui/core/Typography'
+import AddIcon from '@material-ui/icons/Add';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+import Hidden from '@material-ui/core/Hidden';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
 
 import {
   StyledFormControl,
   StyledTextField,
   StyledLabel,
-} from './DirectoryInputs'
-import FieldError from 'components/form/inputs/FieldError'
-import { SingleCheck } from 'components/form/inputs/Check'
+} from './DirectoryInputs';
+import FieldError from 'components/form/inputs/FieldError';
+import { SingleCheck } from 'components/form/inputs/Check';
 
-import { Interpreter, Language } from 'constants/interfaces'
-import { Field, useFormikContext, FieldArray } from 'formik'
+import { Interpreter, Language } from 'constants/interfaces';
+import { Field, useFormikContext, FieldArray, useField } from 'formik';
+import moment from 'moment';
 
 type GridItemInputProps = {
   name: string;
   label: string;
   rows?: any;
-}
+  initialValue?: any;
+};
+
+const setDateFormat = (date?: Date): string => {
```

Comment:
Not sure this should begin with `set`, makes me think it's a property setter. Possibly `formatDate(...)`?

Reply:
FIXED